src/main/java/pl/pszczolkowski/mustdo/domain/task/bo/TasksListBO.java

```diff
@@ -0,0 +1,15 @@
+package pl.pszczolkowski.mustdo.domain.task.bo;
+
+import pl.pszczolkowski.mustdo.domain.task.dto.TasksListSnapshot;
+
+public interface TasksListBO {
+
+   TasksListSnapshot add(String name, Long boardId);
+
+   TasksListSnapshot rename(Long id, String name);
+   
+   void delete(Long id);
+
+   public int removeTasksListsWithBoardId(Long boardId);
```

Comment:
Be consistent - you don't use public modifier in other methods, so don't use it here